scripts/stage-release.sh

```diff
@@ -128,14 +132,18 @@ fi
 echo "Versioning Kroxylicious as ${RELEASE_VERSION}"
 
 mvn -q versions:set -DnewVersion="${RELEASE_VERSION}" -DgenerateBackupPoms=false -DprocessAllModules=true
+
+# Bump version ref in files not controlled by Maven
+${SED} -i -e "s#${CURRENT_VERSION//./\.}#${RELEASE_VERSION}#g" $(find kubernetes-examples -name "*.yaml" -type f)
```

Comment:
what is the bash replacement `${CURRENT_VERSION//./\.}` doing? Is the intent to escape dots for the sed regex input? It doesn't look like it does a replacement for me:

```
[roby@roby kroxylicious]$ echo $CURRENT_VERSION
0.5.0-SNAPSHOT
[roby@roby kroxylicious]$ echo ${CURRENT_VERSION//./\.}
0.5.0-SNAPSHOT
```

might need another escape on the slash:

```
 echo ${CURRENT_VERSION//./\\.}
0\.5\.0-SNAPSHOT
```

Reply:
Yes, you guessed my intent right.  I screwed up the escaping as you suggest.